.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: node_js
-cache: yarn
 node_js:
-  - "0.12"
```

Comment:
😄  Yeah I know right? I think it'll also be good to switch to babel-preset-env, so that the build will just adapt to the environments we want to publish for, instead of converting it all to es5